frame/dapps-staking/src/lib.rs

```diff
@@ -124,6 +125,15 @@ impl<AccountId> DAppInfo<AccountId> {
             state: DAppState::Registered,
         }
     }
+
+    /// `true` if dApp has been unregistered, `false` otherwise
+    fn is_unregistered(&self) -> bool {
+        if let DAppState::Unregistered(_) = self.state {
+            true
+        } else {
+            false
+        }
+    }
 }
```

Comment:
```suggestion
    fn is_unregistered(&self) -> bool {
        std::matches!(self.state, DAppState::Unregistered(_))
    }
}
```

Reply:
I believe `std` may be unavailable for `no_std`.

Reply:
`matches!` is available in `sp-std` so I will just use it from there.

Good suggestion!